CHANGELOG.md

```diff
@@ -10,6 +10,12 @@ and this project adheres to
 
 ### Added
 
+- [#4428](https://github.com/firecracker-microvm/firecracker/pull/4428): Added
```

Comment:
What would be a user noticeable impact from a non-developer point of view?

Reply:
I don't think so. Did you have something particular in mind?

Reply:
nope, I was wondering if we could add something that highlight what acpi support bring to the customers.